src/rest/AnalyticsEndpoint.ts

```diff
@@ -93,22 +95,40 @@ export class AnalyticsEndpoint {
     return this.getFromService<string[]>(url, params);
   }
 
+  public forgetVisitorId() {
```

Comment:
Renamed to `clearCookies`.

It's public since it needs to be called when calling clearLocalData on the [Analytics](../tree/master/src/ui/Analytics/Analytics.ts) component. However, it could easily be redefined over there.
I put it the [AnalyticsEndpoint](../tree/master/src/rest/AnalyticsEndpoint.ts) because that's the only place where `visitorId` and `visitId` are set, and I thought it would be more intuitive if they were removed at the same place.

For now, I haven't moved it over or made it private, but I'm not resolving the conversation since I don't know if it should. What do you think?

Reply:
I discussed the issue with @olamothe, and he said that even though `clearCookies` is public, since it's part of the [AnalyticsEndpoint](../tree/master/src/rest/AnalyticsEndpoint.ts), it should generally speaking not be called by other developers. He also said that since `visitorId` is initialized in it, it also makes sense to clear it from there.